### ISSUE TITLE
Set up credentials service in cloud.gov

### ISSUE THREAD

**Issue by lbeaufort** [fecgov/fecfile-web-api#56]
In order for environment variables to persist when we delete apps, we need to set up a credentials service in cloud.gov. See https://cloud.gov/docs/management/production-ready/#protect-access-to-sensitive-credentials. We should do this for all three apps

### Technical steps
- [x] Create services in all three spaces, bind to apps:
```
cf cups fecfile-api-creds-dev
cf bind-service fecfile-web-api fecfile-api-creds-dev

cf cups fecfile-api-creds-stage
cf bind-service fecfile-web-api fecfile-api-creds-stage

cf cups fecfile-api-creds-prod
cf bind-service fecfile-web-api fecfile-api-creds-prod
```
- [x] change API app settings to use env.get_credential() instead of os.environ() (creds service env vars don't get added to the operating system - for example, `echo $ENV_VAR` won't work if it's in a creds service)
- [x] move non-sensitive config to manifest files
- [X] Generate new secret keys (50 characters)
- [x] move sensitive to creds service except .pem private key value (secret key) (we'll handle private key in the future s3 ticket #110)

`$ ipython`
```
In [1]: from django.core.management.utils import get_random_secret_key   
In [2]: get_random_secret_key()                                                                                                             
Out[2]: ***********redacted
```

follow these instructions to set up json: https://github.com/fecgov/openFEC/wiki/Switch-out-cf-environment-variables

set up dev.env.json (see above)
```
cf uups fecfile-api-creds-dev -p ./dev.env.json 
cf env fecfile-web-api
(optional) cf restage fecfile-web-api
```
- [x] (after merge) unset old env vars not in creds service in dev
```
cf unset-env fecfile-web-api OIDC_RP_CLIENT_ID
cf unset-env fecfile-web-api DJANGO_SECRET_KEY
(rebuild in circle)
```
- [x] (after merge) unset old env vars not in creds service in stage
- [x] (after merge) unset old env vars not in creds service in prod

### Acceptance Criteria ### 

- [x] app environment variable in Cloud.gov persist after app is deleted and recreated.
- [x] new persistence works for web-api dev, stage, and prod
- ~new persistence works for web-app dev, stage, and prod~ we don't currently have server-side environment variables for the web app because by nature it's not able to keep secrets
- [x] for web-api credentials restoration, reference ENV vars in the api Dockerfile

**Comment by mjtravers** [comment]
Ticket passes CR and has been merged into the develop branch. Will wait on moving the ticket into QA until the check box task list in the ticket description has been completed.

**Comment by WiseQA** [comment]
QA after reviewing status of this ticket added Laura to assignee with Matt.  Awaiting verification of task boxes in Acceptance Criteria to be updated.

**Comment by mjtravers** [comment]
For QA: Here's a screenshot of the credential services in cloud.gov:
<img width="787" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMXRwQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--213aae00475ba93b0c55248e184253d1a71b9583/image.png" alt="image.png" />


Passes CR. Sending to QA.
